invokeai/backend/prompting/conditioning.py

```diff
@@ -56,7 +57,7 @@ def get_uc_and_c_and_ec(prompt_string,
     negative_prompt: FlattenedPrompt | Blend = negative_conjunction.prompts[0]
 
     tokens_count = get_max_token_count(model.tokenizer, positive_prompt)
-    if log_tokens or getattr(Globals, "log_tokenization", False):
+    if log_tokens or config.log_tokenization:
```

Comment:
this is different to the `context.services.configuration.log_tokenization` in compel.py - are they both correct or is one preferred, or am i just misunderstanding something?

Reply:
As I understand - in nodes config implemented as service and in this code no node context, so config loads separately by `config = get_invokeai_config()`

Reply:
> this is different to the `context.services.configuration.log_tokenization` in compel.py - are they both correct or is one preferred, or am i just misunderstanding something?

These will end up being the same value. When the context manager starts up it calls `get_invokeai_config()` which will initialize a singleton config object. When conditioning calls that function, it will get the same object. The reason for this is that `conditioning` can get called outside the nodes framework, and won't have access to the context, as StAlKeR noted.